dashboard/test/controllers/data_docs_controller_test.rb

```diff
@@ -43,6 +43,12 @@ class DataDocsControllerTest < ActionController::TestCase
   test_user_gets_response_for :edit, params: -> {@test_params}, user: :teacher, response: :forbidden
   test_user_gets_response_for :edit, params: -> {@test_params}, user: :levelbuilder, response: :success
 
+  # only levelbuilder can edit_all
+  test_user_gets_response_for :edit_all, user: nil, response: :redirect, redirected_to: '/users/sign_in'
+  test_user_gets_response_for :edit_all, user: :student, response: :forbidden
+  test_user_gets_response_for :edit_all, user: :teacher, response: :forbidden
+  test_user_gets_response_for :edit_all, user: :levelbuilder, response: :success
+
```

Comment:
Could you add similar tests for the destroy action?

Reply:
done!